lib/routes/close_channels/coop_closed_channels.dart

```diff
@@ -0,0 +1,272 @@
+    return _breezLib.validateAddress(_addressController.text).then((data) {
+      _log.info("Address validated: $data");
+      _addressValidated = data;
+      return _formKey.currentState.validate();
+    }).catchError((err) {
+      _log.warning("Validating address error", err);
+      _addressValidated = null;
+      return _formKey.currentState.validate();
+    });
+  }
+
+  void _onNext() {
+    _asyncValidate().then((validated) async {
+      if (validated) {
+        final texts = context.texts();
+        final themeData = Theme.of(context);
+
+        _formKey.currentState.save();
+        var validatedAddress = _addressController.text;
+        FocusScope.of(context).requestFocus(FocusNode());
+        promptAreYouSure(
+          context,
+          texts.close_channels_title,
+          RichText(
+            text: TextSpan(
+              style: themeData.dialogTheme.contentTextStyle,
+              text: "${texts.close_channels_confirmation_dialog_message}?\n\n",
+              children: [
+                TextSpan(
+                  text: validatedAddress,
+                  style: themeData.dialogTheme.contentTextStyle.copyWith(
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+              ],
+            ),
+          ),
+          okText: texts.close_channels_confirmation_dialog_action_yes,
+          cancelText: texts.close_channels_confirmation_dialog_action_no,
+          contentPadding: const EdgeInsets.fromLTRB(24, 16, 24, 8),
+        ).then((ok) async {
+          if (ok) {
+            await closeChannels(validatedAddress);
+          }
+        });
+      }
+    }).catchError((error) {
+      _log.warning("onNext error", error);
+      promptError(
+        context,
+        null,
+        Text(
+          extractExceptionMessage(error),
+          style: Theme.of(context).dialogTheme.contentTextStyle,
+        ),
+      );
+    }).whenComplete(() {
+      _log.info("onNext done");
+    });
+  }
+
+  Future<void> closeChannels(String address) async {
+    final texts = context.texts();
+    final navigator = Navigator.of(context);
+    var loaderRoute = createLoaderRoute(
+      context,
+      message: texts.close_channels_loader_message,
+      opacity: 0.8,
+    );
+
+    try {
+      navigator.push(loaderRoute);
+      final closedChannels = await _breezLib.closeChannels(address);
+      if (closedChannels.channels
+          .any((c) => !c.isSkipped || !c.failErr.isNotEmpty)) {
+        _log.info("Closed channels successfully.\n"
+            "List of closed channels: ${closedChannels.channels.toString()}");
+        return;
+      } else {
+        // TODO: Handle the case where all channels are not closed successfully
+        for (var channel in closedChannels.channels) {
+          if (channel.isSkipped) {
+            // Do sth
+          }
+          if (channel.failErr.isNotEmpty) {
+            // Do sth
+          }
+        }
+      }
```

Comment:
> It looks like we are ignoring the result here. It seems that the API returns information on each channel that was closed (or failed). I am only worried if for some reason some of the channels weren't able to be closed so perhaps we should reflect that in the UI. I suggest to involve @kingonly regarding the user experience here.

@roeierez Added placeholder handlers & waiting for feedback on how to move forward.

I also wonder what is the 100% success case here, is it where no channels are skipped and closed with no fail errors? Can there be a good reason to skip channels?